README.md

```diff
@@ -47,86 +44,77 @@ to model the relationships between different components.
 
 2. Set these environment variables for your AWS access credentials:
 
-  ~~~
+```bash
   export AWS_ACCESS_KEY_ID=your_access_key_id
   export AWS_SECRET_ACCESS_KEY=your_secret_access_key
-  ~~~
+ ```
 
-  Alternatively, you can place the credentials in a file at
-`~/.aws/credentials` based on the following template:
+  Alternatively, you can place the credentials in a file at '~/.aws/credentials' based on the following template:
 
-  ~~~
+```bash
  [default]
   aws_access_key_id = your_access_key_id
   aws_secret_access_key = your_secret_access_key
-  ~~~
+```
 
   If you have Puppet running on AWS, and you're running the module examples, you can instead use [IAM](http://aws.amazon.com/iam/). To do this, assign the correct role to the instance from which you're running the examples. For a sample profile with all the required permissions, see the [IAM profile example](https://github.com/puppetlabs/puppetlabs-aws/tree/master/examples/iam-profile/).
 
 3. Finally, install the module with:
 
-~~~
+```bash
 puppet module install puppetlabs-aws
-~~~
+```
 
 #### A note on regions
 
-By default the module looks through all regions in AWS when
-determining if something is available. This can be a little slow. If you
-know what you're doing you can speed things up by targeting a single
-region using an environment variable.
+By default the module looks through all regions in AWS when determining if something is available. This can be a little slow. If you know what you're doing you can speed things up by targeting a single region using an environment variable.
 
-~~~
+```bash
 export AWS_REGION=eu-west-1
-~~~
+````
 
 #### A note on proxies
 
-By default the module accesses the AWS API directly, but if you're in an
-environment which doesn't have direct access you can provide a proxy
-setting for all traffic like so:
+By default the module accesses the AWS API directly, but if you're in an environment which doesn't have direct access you can provide a proxy setting for all traffic like so:
 
-~~~
+```bash
 export PUPPET_AWS_PROXY=http://localhost:8888
-~~~
+```
 
 #### Configuring the aws module using an ini file
 
-The AWS region and HTTP proxy can be provided in a file called
-`puppetlabs_aws_configuration.ini` in the Puppet confdir
-(`$settings::confdir`) using this format:
-
+The AWS region and HTTP proxy can be provided in a file called 'puppetlabs_aws_configuration.ini' in the Puppet confdir ('$settings::confdir') using this format:
+```ini
     [default]
       region = us-east-1
       http_proxy = http://proxy.example.com:80
-
+```
 ##Getting Started with aws
 
-The aws module allows you to manage AWS using the Puppet DSL. To stand up an instance with AWS, use the `ec2_instance` type. The following code sets up a very basic instance:
+The aws module allows you to manage AWS using the Puppet DSL. To stand up an instance with AWS, use the 'ec2_instance' type. The following code sets up a very basic instance:
```

Comment:
Because this refers to a type, it should be in backticks. :-( (Types, classes, defined types, functions are pieces of code and set in backticks.)